src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -27,7 +27,7 @@ public class AddCommand extends Command {
             + "[" + PREFIX_PHONE + "PHONE] "
             + "[" + PREFIX_EMAIL + "EMAIL] "
             + "[" + PREFIX_ADDRESS + "ADDRESS] "
-            + "[" + PREFIX_LANGUAGE + "LANGUAGE]...\n"
+            + "[" + PREFIX_LANGUAGE + "LANGUAGE"
             + "[" + PREFIX_TAG + "TAG]...\n"
```

Comment:
Might I ask why the `...` were removed for languages here and under the EditCommand.java files? They indicate that the item can be used multiple times, as mentioned in the user guide: 
https://ay2223s2-cs2103t-t12-4.github.io/tp/UserGuide.html
Perhaps I misunderstood the intention here?

I believe the first `\n` could be removed safely though, I think that was my mistake.

Reply:
Opps, i think i just now delete away too much

Reply:
Just referring to the User Guide here, since we're using the same formatting:

> Items with `…​` after them can be used multiple times including zero times.
> e.g. `[t/TAG]…​` can be used as   (i.e. 0 times), `t/friend`, `t/friend t/family` etc.

So the individual `...` do not refer to the line as a whole being extended, rather, it indicated that the individual items may occur multiple times, and I believe it should be present behind both